src/validations/order.validation.ts

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi';
+
+export const orderSchema = {
+  order: Joi.object({
+    address: Joi.string().min(1).max(255).required().messages({
+      'string.empty': 'Address cannot be an empty field',
+      'string.min': 'Address must contain at least 1 character',
+      'string.max': 'Address must contain at most 255 characters',
+      'any.required': 'Address is a required field',
+    }),
+    country: Joi.string().min(1).required().messages({
+      'string.empty': 'Country cannot be an empty field',
+      'string.min': 'Country must contain at least 1 character',
+      'any.required': 'Country is a required field',
+    }),
+    city: Joi.string().min(1).required().messages({
+      'string.empty': 'city name cannot be an empty field',
+      'string.min': 'city name must contain at least 1 character',
+      'any.required': 'city name is a required field',
+    }),
+    zipCode: Joi.string().min(1).required().messages({
+      'string.empty': 'zipCode name cannot be an empty field',
+      'string.min': 'zipCode name must contain at least 1 character',
+      'any.required': 'zipCode name is a required field',
+    }),
+    expectedDeliveryDate: Joi.string().min(1).required().messages({
+      'string.empty': 'expectedDeliveryDate name cannot be an empty field',
+      'string.min':
+        'expectedDeliveryDate name must contain at least 1 character',
+      'any.required': 'expectedDeliveryDate name is a required field',

```

Comment:
You don't have to specify a message for every violation. In most cases Joi returns the appropriate message